<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_desconectar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Desconectar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_led_on">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>LedOn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_led_Off">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>LedOff</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>241</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_GETID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>GetID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Usuario :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Clave : </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>135</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="le_usuario">
      <property name="text">
       <string>qt</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_clave">
      <property name="text">
       <string>esp32esp32</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>66</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Servidor :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Puerto :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>135</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="le_servidor">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>192.168.0.141</string>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_puerto">
      <property name="text">
       <string>1883</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Lista de Dispositivos:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListDispo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="grafico" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="boxdispositivos">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
